src/utils/tokens.js

```diff
@@ -0,0 +1,47 @@
+
+import sendJson from '../tmp_fetch_send_json'
+import { ACCOUNT_HELPER_URL } from './wallet'
+import * as Sentry from '@sentry/browser'
+
+export const getLikelyContracts = async (accountId) => {
+    try {
+        return await sendJson('GET', `${ACCOUNT_HELPER_URL}/account/${accountId}/likelyTokens`)
+    } catch (e) {
+        logError(e);
+    }
+}
+
+export const getMetadata = async (contract, account) => {
+    let metadata
+
+    try {
+        metadata = await account.viewFunction(contract, 'ft_metadata')
+    } catch (e) {
+        logError(e);
+    } finally {
+        return {
+            contract,
+            metadata
+        }
+    }
+}
+
+export const getBalanceOf = async (contract, account, accountId) => {
```

Comment:
nit: seems like you can avoid some boilerplate:

```js
const logAndIgnoreError = (promise) => promise.catch(error => console.warn(error));

export const getBalanceOf = async (contract, account, accountId) => {
  let balance = await logAndIgnoreError(account.viewFunction(contract, 'ft_balance_of', { account_id: accountId }))
  return { contract, balance }
}
```

Reply:
Good idea, done